updating/updating-cluster-cli.adoc

```diff
@@ -0,0 +1,26 @@
+[id="updating-cluster-cli"]
+= Updating a cluster to a minor version by using the CLI
+include::modules/common-attributes.adoc[]
+:context: updating-cluster-cli
+
+toc::[]
+
+You can update, or upgrade, an {product-title} cluster by using the
+OpenShift CLI (`oc`).
+
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited in the
+following ways:
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 3 to OpenShift 4 Beta Drop 4.
+* The ability to update from a beta drop of {product-title} to a full version of
+{product-title} is not guaranteed.
+* After you install OpenShift 4 Beta Drop 4, you can update your cluster to
+one of the available nightly builds.
```

Comment:
The note's just there for the beta, and I will remove it for GA. The last I heard was that you can update from Beta 7 (not 5 anymore - I need to update the note) to the GA version by using the stable channel.

Reply:
If Beta 7 is our final beta build, then we should on the same page, only support Beta 7 to GA upgrade.

Reply:
About 'nightly build', even this is for beta user, but as a customer, I do not think they know anything about nightly build. Nightly build should be only internal usage, am I wrong?

Reply:
> "You cannot update between different beta drops, such as from OpenShift 4 Beta Drop 3 to OpenShift 4 Beta Drop 4"

> "After you install OpenShift 4 Beta Drop 4, you can update your cluster to one of the available nightly builds."

Per my understanding, the 1st sentence was saying upgrade between all different versions of beta is not supported, beta 3 -> beta4 is not supported, beta 5 -> beta 6 is not supported either. While the 2nd sentence was saying user can do any upgrade since beta 4. They seem like conflicting.

Reply:
> Nightly build should be only internal testing usage, am I wrong?

There's talk of getting some/all nightlies out where customers can pull them, but yeah, at the moment there is no external access to nightlies.